test/functional/rpc_createmultisig.py

```diff
@@ -18,15 +18,19 @@
     assert_equal,
 )
 from test_framework.wallet_util import bytes_to_wif
+from test_framework.wallet import (
+    MiniWallet,
+    address_to_scriptpubkey,
+    getnewdestination,
+)
 
 class RpcCreateMultiSigTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 3
         self.supports_cli = False
-
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
+        if self.is_bdb_compiled():
+            self.requires_wallet = True
```

Comment:
Why is this needed? IIRC this will only import the coinbase keys, but they aren't needed?

Reply:
This is actually needed to create default wallets. Without this `check_addmultisigaddress_errors()` fails.